inc/saiswitch.h

```diff
@@ -2350,6 +2371,28 @@ typedef enum _sai_switch_attr_t
      */
     SAI_SWITCH_ATTR_MACSEC_OBJECT_LIST,
 
+    /**
+     * @brief IPsec object for this switch.
+     *
+     * @type sai_object_id_t
+     * @flags CREATE_AND_SET
+     * @objects SAI_OBJECT_TYPE_IPSEC
+     * @allownull true
+     * @default SAI_NULL_OBJECT_ID
+     */
+    SAI_SWITCH_ATTR_IPSEC_OBJECT_ID,
+
+    /**
+     * @brief TPID in IPsec SA-Tag.  This is used only between a Switch ASIC
+     * and IPsec-enabled PHY chips and not packets on external network.
+     *
+     * @type sai_uint16_t
+     * @flags CREATE_AND_SET
+     * @isvlan false
+     * @default 0x1234
```

Comment:
is this actual real default value ?

Reply:
Changing this to 0xFFFE. This is a reserved EtherType.  This is only used internal to the switch - for communication between the Switch ASIC and the PHY/Gearbox.